src/styles/v2.css

```diff
@@ -0,0 +1,31 @@
+:root {
+  --dsdl-white: #fff;
+  --dsdl-black: #101010;
+}
```

Comment:
i think these are generically useful enough that it would make sense to move them into calitp.org itself, but at a minimium i figured we can/should talk about names in this PR.